src/components/Image/Image.jsx

```diff
@@ -0,0 +1,36 @@
+import { PropTypes } from 'prop-types';
+import styled from '@emotion/styled';
+
+const Image = ({ src, alt, width, height, type, mode }) => {
+  const { PUBLIC_URL } = process.env;
+  const ProfileDefaultImage = `${PUBLIC_URL}/images/profile-image-default.png`;
+  const PostDefaultImage = `${PUBLIC_URL}/images/post-image-default.jpg`;
+
+  return (
+    <StyledImage
+      src={src || (type == 'profile' && ProfileDefaultImage) || PostDefaultImage}
+      alt={alt || '프로필 이미지'}
+      width={width}
+      height={height}
+      type={type}
+      mode={mode}
+    />
+  );
+};
+
+const StyledImage = styled.img`
+  width: ${({ width, type }) => width || (type === 'profile' && '4.5rem') || '20rem'};
+  height: ${({ height, type }) => height || (type === 'profile' && '4.5rem') || '20rem'};
+  object-fit: ${({ mode }) => mode || 'cover'};
+`;
+
+Image.propTypes = {
+  src: PropTypes.string.isRequired,
+  alt: PropTypes.string.isRequired,
```

Comment:
테스트를 해보니까 undefined prop이 입력되었을 때 기본 이미지는 출력되는데 undefined 값을 받았다는 warning이 떠서 원래 하셨던 대로 해주시면 될 것 같습니다!

```suggestion
  src: PropTypes.string,
  alt: PropTypes.string,
```